### PR TITLE
feat(reusable): Create a reusable component that contains the titles and subtitles

### PR DESCRIPTION
**Pull request by Keikonichi**
Create a reusable component that contains the titles and subtitles of the pages.

image

image

### CODE REVIEW

**Comment by Keikonichi** [comment]
I took the opportunity to create the text that appears just below the page title.

image

@thiagobraga 
@renatoagomes

File: app/projetos/page.tsx [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+      {boxData.map(function (item) {
+        return (
+          <PageTitle
+            subtitle1={item.subtitle1}
+            subtitleClass={item.subtitleClass}
+            title={item.title}
+            key={item.key}
+          />
+        );
+      })}
```
Comment (thiagobraga) [inline app/projetos/page.tsx]: ### Suggestion

```jsx
{boxData.map((item) => (
  <PageTitle
    subtitle1={item.subtitle1}
    subtitleClass={item.subtitleClass}
    title={item.title}
    key={item.key}
  />
))}
```

File: app/servicos/page.tsx [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+    title: 'Nossos Serviços',
+    key: 1,
+  },
+];
+
 export default function Services() {
   return (
     <>
+      <div>
+        {boxData.map(function (item) {
```
Comment (thiagobraga) [inline app/servicos/page.tsx]: The same above, use an arrow function

File: app/components/PageTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+interface PageTitleProps {
+  subtitle1?: string;
+  subtitle2?: string;
+  subtitle3?: string;
+  subtitleClass?: string;
+  title?: string;
+}
+
+const PageTitle = (props: PageTitleProps) => {
```
Comment (thiagobraga) [inline app/components/PageTitle.tsx]: Just a suggestion, I would keep components definition the same way you did below in PreContentText

```jsx
export default function PageTitle(props: PageTitleProps) {
  return (
    ...
  );
}
```

**Review by thiagobraga — COMMENTED** [review]
@Keikonichi Nice work. I left some comments. Tks

**Comment by Keikonichi** [comment]
Made the suggested changes.

@thiagobraga 
@renatoagomes

**Comment by thiagobraga** [comment]
@Keikonichi Please resolve the conflicts because I merged other PRs that may changed some files of this PR.

**Comment by Keikonichi** [comment]
I did resolve conflicts.

@thiagobraga 
@renatoagomes

**Pull request merged**